pkg/api/types.go

```diff
@@ -2103,6 +2103,16 @@ type PodSpec struct {
 	// file if specified. This is only valid for non-hostNetwork pods.
 	// +optional
 	HostAliases []HostAlias
+	// If specified, indicates the pod's priority. "system" is a special keyword
+	// which indicates the highest priority. Any other name must be defined by
+	// creating a PriorityClass object with that name.
+	// If not specified, the pod priority will be zero.
+	// +optional
+	PriorityClassName string
+	// The priority value which is resolved by the Admission Controller from
+	// PriorityClassName. The higher the value, the higher the priority.
+	// It must not be specified by user.
+	Priority *int32
```

Comment:
Admission controller rejects a pod if its `Priority` is set by user. I have already implemented it, but since those are branched from this PR, I cannot send them for review yet.